bigchaindb/backend/connection.py

```diff
@@ -0,0 +1,15 @@
+from importlib import import_module
+
+BACKENDS = {
+    'rethinkdb': 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'
+}
+
+
+def connect(backend, host, port, dbname):
```

Comment:
> What do you think about making this a single dispatched call (that just returns a Connection class) and moving the import of the backend module to be an explicit part of the initialization phase?

we discussed this in person, but as a quick answer: it's not feasible, we need to instantiate the concrete class somewhere.



> Erroring out immediately if we can't find the backend module protects us from bad configs and also makes it easier to accept backend plugins in the future (I stress that I think this would be very easy, but we can save it for when we feel we're ready ). I'm sure it'll still error out like this right now, but it's not very obvious.

That's a good point, as discussed I can wrap it in a `try ... except` and raise a `sys.exit` for now, then we can iterate again and make it better.